sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ Thank you to our developer community members who helped to make the Event Hubs c
 
 - When the `EventProcessorClient` detects a partition being stolen outside of a load balancing cycle, it will immediately surrender ownership rather than waiting for a load balancing cycle to confirm the ownership change.  This will help reduce event duplication from overlapping ownership of processors.
 
+- The `ConnectionOptions` available when creating a processor now supports registering a callback delegate for participating in the validation of SSL certificates when connections are established.  This delegate may override the built-in validation and allow or deny certificates based on application-specific logic.
```

Comment:
```suggestion
- The `ConnectionOptions` available when creating a processor now support registering a callback delegate for participating in the validation of SSL certificates when connections are established.  This delegate may override the built-in validation and allow or deny certificates based on application-specific logic.
```

Reply:
The funny part is I started in the singular and then flip-flopped a few times.

Reply:
I looked at it and squinted my eyes a few times and I landed on singular being correct since the FooOptions is a singular entity even though it ends in s 😄